WordPress/gutenberg
# Table block: Fix focus loss after Create Table button is selected

## What?


The PR fixes focus on table creation.

## Why?


When focus loss occurs after table creation, users may be confused.

## How?


After the table creation function executes, focus is placed in the table.

## Testing Instructions





1. Open post/page.
2. Insert a table block.
3. Select Create Table button.
4. You should notice that focus is placed in the table cell.

## Screenshots or screencast
